src/components/datasets/DatasetOverview/DatasetOverview.vue

```diff
@@ -337,6 +337,15 @@
           >
             Save
           </button>
+          <button
+            v-else
+            slot="title-aux"
+            class="linked-9"
+            :disabled="datasetLocked"
+            @click="isEditingMarkdown2 = true"
+          >
+            Update
+          </button>
```

Comment:
@annaleeswitek I don't think this button existed.
https://github.com/Pennsieve/pennsieve-app/commit/f9a9c63af20f91da2ddd47c7a9d97f153a3f3e23#diff-21f41a35951c78e571bee65d154aead99b30f4681ef1000394f93a16747674a4R332

should we be doing anything further to confirm if the user is a publisher?

Reply:
@edmore - thanks for tracking down that diff from when the component was initially created. One question I have is if the button never existed, then why is there `isEditingMarkdown2` functionality - it would be inaccessible without the button (the button toggles the markdown editor open and closed). 

In any case, adding the button in is the correct path toward getting this feature functional, so we should proceed with adding it back in. 

As for checking is the user is a publisher - I will look into that and update everyone here with my findings.

Reply:
@edmore The logic, as I understand it from speaking with Joost, is supposed to be something like this: 

Publishers should be the only users who can edit the the changelog field **after** the dataset is locked for publishing. There is a property on the FE, `datasetLocked` that is controlling whether the `Update` button on the changelog field is enabled or disabled. Currently, the logic is: `button is disabled when datasetLocked === true`

However, yesterday Joost suggested it was possible that the `datasetLocked` property is not reflecting the correct state (i.e. the dataset should be locked but datasetLocked is false). 

My next step is going to be to figure out if that's the issue here.